internal/bminventory/inventory.go

```diff
@@ -2353,18 +2645,170 @@ func (b *bareMetalInventory) updateClusterData(_ context.Context, cluster *commo
 		updates["disk_encryption_mode"] = params.ClusterUpdateParams.DiskEncryption.Mode
 	}
 
-	if len(updates) > 0 {
-		updates["trigger_monitor_timestamp"] = time.Now()
-		dbReply := db.Model(&common.Cluster{}).Where("id = ?", cluster.ID.String()).Updates(updates)
-		if dbReply.Error != nil {
-			return common.NewApiError(http.StatusInternalServerError, errors.Wrapf(err, "failed to update cluster: %s", params.ClusterID))
+	if len(updates) > 0 {
+		updates["trigger_monitor_timestamp"] = time.Now()
+		dbReply := db.Model(&common.Cluster{}).Where("id = ?", cluster.ID.String()).Updates(updates)
+		if dbReply.Error != nil {
+			return common.NewApiError(http.StatusInternalServerError, errors.Wrapf(err, "failed to update cluster: %s", params.ClusterID))
+		}
+	}
+
+	return nil
+}
+
+func (b *bareMetalInventory) v2UpdateClusterData(_ context.Context, cluster *common.Cluster, params installer.V2UpdateClusterParams, usages map[string]models.Usage, db *gorm.DB, log logrus.FieldLogger, interactivity Interactivity) error {
```

Comment:
the same comment regarding DiskEncryption as above

Reply:
Fixed